lib/modules/manager/kustomize/extract.ts

```diff
@@ -144,11 +144,18 @@ export function extractHelmChart(
     return null;
   }
 
+  let datasrc = HelmDatasource.id;
+  let registryUrl = helmChart.repo;
+  if (registryUrl.startsWith('oci://')) {
+    datasrc = DockerDatasource.id;
+    registryUrl = registryUrl.replace('oci://', 'https://');
+  }
```

Comment:
return here without registry url.
remove oci prefix from registry url and join it with depName. then return it as packageName.

checkout helm v3 manager for samples